packages/app/src/renderer/features/current-game-overview.tsx

```diff
@@ -64,30 +67,41 @@ const getFactionMatrix = (gameData: GameData): string => {
 };
 
 const CurrentGameOverview: React.FC<Props> = ({ game }) => {
-  const [mapApiData, setMapApiData] = useState(null);
+  const dispatch = useDispatch();
+  const applicationCache = useSelector(selectCache);
+  const [requestState, setRequestState] = useState<"loading" | "idle" | "completed">("idle");
   const [mapWinLosses, setMapWinLosses] = useState<{ wins: number; losses: number }>();
   const [mapFound, setMapFound] = useState(false);
 
-  // listen map stats from firestore
   useEffect(() => {
-    //convertDateToMonthTimestamp(new Date())
-    const monthTimestamp =
-      Date.UTC(new Date().getUTCFullYear(), new Date().getUTCMonth() - 1, 1) / 1000;
-    //Date.UTC(new Date().getUTCFullYear(), new Date().getUTCMonth() - 1, 1) / 1000;
-    onSnapshot(doc(getFirestore(), `stats/month/${monthTimestamp}/`, "mapStats"), (doc) => {
-      setMapApiData(doc.data());
-    });
-  }, []);
-
-  useEffect(() => {
-    events.game_displayed();
-  }, [game]);
-
-  // update displayed map stats when gamedata changes
-  useEffect(() => {
-    // can only show valid data classic games
-    if (mapApiData) {
-      const result = findMapInApiData(mapApiData, game);
+    if (
+      !applicationCache.mapStats ||
```

Comment:
@JohannesMerkt  Hm can you just double check that this is correct? So `applicationCache` is object {} and we are checking if it has key mapStats right? 

I think it's correct.

Reply:
This is correct. It will be undefined if it is a empty object